### PR TITLE
v7.5 supports Rocky Linux 9.1

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

TiDB v7.5 supports Rocky Linux 9.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: hardware-and-software-requirements.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ TiDB 作为一款开源一栈式实时 HTAP 数据库，可以很好地部署和
 |  Debian 9 (Stretch) 及以上的版本  |  x86_64           |
 |  Fedora 35 及以上的版本   |  x86_64           |
 |  openSUSE Leap 15.3 以上的版本（不包含 Tumbleweed） |  x86_64           |
+|  Rocky Linux 9.0  |  <ul><li>x86_64</li><li>ARM 64</li></ul> |
```
Comment (jebter) [inline hardware-and-software-requirements.md:35]: 9.1 及以上的版本

**Review by jebter — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Comment by shawn0915** [comment]
图片

提个小建议，这展示顺序能不能调理一下？现在有点跳跃

**Comment by Oreoxmt** [comment]
> 图片
> 
> 提个小建议，这展示顺序能不能调理一下？现在有点跳跃

因为 LTS 版本针对不同操作系统提供了两种级别质量标准的支持：可满足企业级生产质量的要求和不保障企业级生产质量要求，对应两个表格，详见 https://docs.pingcap.com/zh/tidb/stable/hardware-and-software-requirements

所以 https://docs.pingcap.com/zh/tidb/dev/hardware-and-software-requirements 文档中表格的顺序逻辑是：可满足企业级生产质量的要求 -> 不保障企业级生产质量要求。

在相同的质量支持标准内，我们会优化一下不同操作系统的顺序

**Pull request merged**